src/desdemona/query.clj

```diff
@@ -11,22 +13,36 @@
   The n-answers is simply passed to core.logic/run; we're relying on
   it to correctly bound the number of answers. This helps us limit how
   long it takes to query."
-  [n-answers query events]
+  [n-answers logic-query events]
   `(l/run ~n-answers [results#]
      (l/fresh [~'x] ;; ~'x means "literally x, don't gensym", see #28
        (l/== [~'x] results#)
        (l/membero ~'x ~events)
-       ~query)))
+       ~logic-query)))
 
-(defn run-query
+(defn run-logic-query
   "Runs a query over some events and finds n answers (default 1)."
-  ([query events]
-   (run-query 1 query events))
-  ([n-answers query events]
-   (let [compiled-query (gen-query n-answers query events)
-         old-ns *ns*]
+  ([logic-query events]
+   (run-logic-query 1 logic-query events))
+  ([n-answers logic-query events]
+   (let [old-ns *ns*]
      (try
        (in-ns 'desdemona.query)
-       (eval compiled-query)
+       (eval (generate-logic-query n-answers logic-query events))
```

Comment:
Is the benefit of moving this here instead of in the `let` above that it's more lazily evaluated? That it's actually executed inside the right namespace? Or is the functionality identical and it's just for organization?